doc/source/getting_started/launcher.rst

```diff
@@ -0,0 +1,228 @@
+
+
+This is the easiest and fastest way to get PyMAPDL up and running. 
+But you need to be able to launch MAPDL locally.
+
+If PyMAPDL cannot find your local installation, please refer to
+`Setting MAPDL location in PyMAPDL`_.
+
+For more information on controlling how MAPDL launches locally, see the
+description of the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` function.
+
+
+Connect PyMAPDL to a local MAPDL instance
+-----------------------------------------
+
+.. _launch_grpc_madpl_session:
+
+Launch a local gRPC MAPDL session
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can start MAPDL from the command line and then connect to it.
+
+To launch MAPDL on Windows (assuming a :file:`C:/Program Files/ANSYS Inc/v241` installation), use:
+
+.. code:: pwsh-session
+
+    C:/Program Files/ANSYS Inc/v241/ansys/bin/winx64/ANSYS211.exe -grpc
+
+To launch MAPDL on Linux (assuming ANSYS is installed in :file:`/usr/ansys_inc` installation), use:
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -grpc
+
+This starts up MAPDL in gRPC mode, and MAPDL should output:
+
+.. code:: output
+
+     Start GRPC Server
+
+     ##############################
+     ### START GRPC SERVER      ###
+     ##############################
+
+     Server Executable   : MapdlGrpc Server
+     Server listening on : 0.0.0.0:50052
+
+You can configure the port that MAPDL starts on with the ``-port`` argument.
+For example, you can start the server to listen for connections at 
+port 50005 with:
+
+.. code:: console
+
+    /usr/ansys_inc/v241/ansys/bin/ansys211 -port 50005 -grpc
+
+
+.. _connect_grpc_madpl_session:
+
+Connect to the local MAPDL instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An MAPDL gRPC server can be connected to from the same host by using:
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import Mapdl
+    >>> mapdl = Mapdl()
+
+This assumes that your MAPDL service is running locally on the default IP address 
+(``127.0.0.1``) and on the default port (``50052``).
+
+You can also use the :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>` to connect to an already launched MAPDL instance by setting the ``start_instance`` argument to ``False``.
+
+.. code:: pycon
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> mapdl = launch_mapdl(start_instance=False)
+
+If you are connecting to an MAPDL Docker image, the procedure is the same.
+Just make sure that you specify the mapped port instead of the internal MAPDL port.
+For more information, see :ref:`pymapdl_docker`.
+
+
+.. _connect_grpc_remote_madpl_session:
+
+Connect PyMAPDL to a remote MAPDL instance
+------------------------------------------
+
+If you want to connect to a **remote** MAPDL instance, you must know the IP 
+address of that instance, so you can connect to it.
+For example, if on your local network at IP address ``192.168.0.1`` there is a
+computer running MAPDL on the port 50052, you can connect to it with:
```

Comment:
```suggestion
computer running MAPDL on the port ``50052``, you can connect to it with:
```

Reply:
@germa89 @PipKat

For a python library I think that "code" means arbitrary python code. "command" has a specific meaning for PyMAPDL, which is one of the many MAPDL commands listed in its command reference. We should not use the term "command" for any other purpose.

Reply:
@koubaa Interesting point. Do you know even want to use command for procedures outside of PyMAPDL, like during installation and setup? I can certainly see how once you are using PyMAPDL the use of the word command gets confusing. I'll asking to seek knowledge--because I generally use command in PyAnsys libraries when not introducing several lines of code in a script. I could be suggesting bad edits everywhere!

Reply:
@PipKat good point.  FYI : @germa89
 
 Terminals also have their own concept of a command. So there are two flavors of commands in PyMAPDL - the MAPDL command and the terminal command.  So I will revise my above recommendation to always use the term "code" for python code, and always qualify the term "command" as "MAPDL command" if it is part of python code. For terminal commands, you can use the unqualified "command" or the qualified "terminal command", though I think the unqualified is usually self-explanatory